lib/active_model/serializable_resource.rb

```diff
@@ -30,11 +30,19 @@ def serialization_scope_name=(scope_name)
       serializer_opts[:scope_name] = scope_name
     end
 
+    # NOTE: if no adapter is available, returns the resource itself.
     def adapter
```

Comment:
well, it is and it isn't.  It's essentially a no-op adapter but without a special object of that name wrapping it :)